pkg/cmd/limits/limits.go

```diff
@@ -0,0 +1,51 @@
+package limits
+
+import (
+	"log/slog"
+
+	"github.com/KimMachineGun/automemlimit/memlimit"
+	"github.com/jzelinskie/cobrautil/v2"
+	"github.com/rs/zerolog"
+	slogzerolog "github.com/samber/slog-zerolog/v2"
+	"github.com/spf13/cobra"
+	"go.uber.org/automaxprocs/maxprocs"
+
+	log "github.com/authzed/spicedb/internal/logging"
+)
+
+// SetLimitsRunE wraps the RunFunc with setup logic for memory limits and maxprocs
+// limits for the go process. Implementing it here means that we can use already-
+// established loggers and wrap only those commands that require it.
+func SetLimitsRunE() cobrautil.CobraRunFunc {
```

Comment:
This seems useful enough to put into `cobrautil` so it can be used across other projects.